bin/git-repl

```diff
@@ -2,24 +2,29 @@
 
 git version
 echo "git-extras version ""$(git-extras -v)"
-echo "type 'ls' to ls files below current directory, '!command' to execute any command or just 'subcommand' to execute any git subcommand"
+echo "type 'ls' to ls files below current directory, '!command' to execute any command or just 'subcommand' to execute any git subcommand. 'quit', 'exit', 'q', 'x', ^D, or ^C to exit the git repl."
 
 while true; do
   # Current branch
   cur=$(git symbolic-ref HEAD 2> /dev/null | cut -d/ -f3-)
 
   # Prompt
   if test -n "$cur"; then
-    prompt="git ($cur)> "
+    prompt="$exit_status|git ($cur)> "
```

Comment:
`0|git ` is not so obvious. Maybe we can omit the `0` and only show non-zero exit code like this `exit code: 1 | git ...`? Any idea to make the exit code be more readable is welcome.

Reply:
Yeah, there are a lot of different ways it could be displayed. I have my bashrc set up to display the return code in red followed by a ⏎ (a unicode character which none of my terminals ever display correctly).
![image](https://github.com/user-attachments/assets/b749251c-e499-4e22-84fd-ea6088c455a4)
Fish shell seems to display the code in red in the prompt in brackets, if the return code isn't 0.
![image](https://github.com/user-attachments/assets/90c5f4b1-f376-4e8a-9a19-810379be4c82)
A friend of mine, if memory serves, has his shell configured to print a line after every process that exits non-zero with something like `12 ← the command exited with this status` highlighted in red. (I don't have a screenshot of this.)

Reply:
@wyattscarpenter 
Maybe we can try the fish shell style?

Reply:
@spacewander happy to! Do you want it colored? (Everything in the current git repl is monochrome, but theoretically there shouldn't be any problem adding the terminal escape code for the color, which should be supported everywhere.)

Reply:
@wyattscarpenter 
Adding color is great! Let's do it.